### PR TITLE
feat(priority-alerts): Change the default alert condition to NewHighPriorityIssueCondition

### PR DESCRIPTION
**Pull request by snigdhas**
The default alert condition for a new issue alert will now include both the `NewHighPriorityIssueCondition` and `ExistingHighPriorityIssueCondition` conditions for organizations where the feature is enabled. 

image

### CODE REVIEW

File: static/app/types/alerts.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const enum IssueAlertConditionType {
   EVENT_FREQUENCY = 'sentry.rules.conditions.event_frequency.EventFrequencyCondition',
   EVENT_UNIQUE_USER_FREQUENCY = 'sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition',
   EVENT_FREQUENCY_PERCENT = 'sentry.rules.conditions.event_frequency.EventFrequencyPercentCondition',
-  NEW_HIGH_PRIORITY_ISSUE = 'sentry.rules.conditions.high_priority_issue.NewIssueCondition',
```
Comment (scefali) [inline static/app/types/alerts.tsx:42]: @snigdhas do we have these in the wild already? Won't such a change break things?

Comment (snigdhas) [reply static/app/types/alerts.tsx:42]: This was a typo, but the constant isn't used anywhere so it should be okay

**Comment by schew2381** [comment]
I believe the string may be incorrect [here](https://github.com/getsentry/sentry/blob/4a7adbbac855e04b89ea59eb733c8e5c3234c192/src/sentry/constants.py#L269), it has `new_high_priority_issue` vs `high_priority_issue` as specified in the [condition class](https://github.com/getsentry/sentry/blob/4ba0cf2a048f1b47941d793a88ebe0dc0ef47601/src/sentry/rules/conditions/new_high_priority_issue.py#L18)

**Comment by snigdhas** [comment]
Great catch, ty @schew2381!

**Review by scttcper — APPROVED** [review]

**Pull request merged**